observability/opentelemetry/src/main/java/org/wildfly/extension/opentelemetry/OpenTelemetrySubsystemRegistrar.java

```diff
@@ -56,22 +55,19 @@
  */
 
 class OpenTelemetrySubsystemRegistrar implements SubsystemResourceDefinitionRegistrar, ResourceServiceConfigurator {
-    public static final boolean IS_WILDFLY_PREVIEW;
-
-    static {
-        // Temporary check to determine if the server is WildFly Standard or WildFly Preview. This is
-        // a *temporary* check, so there should be no long-term dependencies on this. -- jdl
-        boolean classFound = false;
-        try {
-            Class.forName("io.smallrye.opentelemetry.implementation.exporters.metrics.VertxMetricsExporterProvider");
-            classFound = true;
-        } catch (ClassNotFoundException e) {
-        }
-        IS_WILDFLY_PREVIEW = classFound;
-    }
 
     public static final String API_MODULE = "org.wildfly.extension.opentelemetry-api";
-    public static final List<String> EXPORTED_MODULES = new ArrayList<>();
+    public static final List<String> EXPORTED_MODULES = List.of(
+        "io.opentelemetry.api",
+        "io.opentelemetry.api.events",
+        "io.opentelemetry.context",
+        "io.opentelemetry.exporter",
+        "io.opentelemetry.otlp",
+        "io.opentelemetry.sdk",
+        "io.opentelemetry.semconv",
+        "io.smallrye.opentelemetry",
+        "io.opentelemetry.instrumentation.api",
+        "io.opentelemetry.semconv");
```

Comment:
This seems to be duplication with line 67, can you please double-check?

Reply:
It is. Good eye. Fixed.